jsl/document.py

```diff
@@ -15,11 +15,19 @@ def _set_owner_to_document_fields(cls):
             field.owner_cls = cls
 
 
+# INHERITANCE CONSTANTS AND MAPPING
+
+INLINE = 'inline'  # default inheritance mode
 ALL_OF = 'all_of'
-"""All-of inheritance mode"""
+ANY_OF = 'any_of'
+ONE_OF = 'one_of'
 
-INLINE = 'inline'
-"""Inline (default) inheritance mode"""
+_INHERITANCE_MODES = {
+    INLINE: 'allOf',  # used in the case that an inline class inherits from non-inlined bases
```

Comment:
This mapping for the INLINE mode was included for backwards compatibility. Inline classes which have parents are not inlined in the current build (e.g. https://github.com/aromanovich/jsl/blob/master/tests/test_inheritance.py#L105-L120).